Improve docs metadata (#405)
o1-labs/docs2: docs/zkapps/how-to-write-a-zkapp.mdx

---
title: How to Write a zkApp
hide_title: true

> description: A comprehensive guide on writing a zkApp which includes smart contracts and a UI for interacting with them. This guide will walk you through the process from installation to creating smart contracts with SnarkyJS for Mina Protocol.

Reviewer: ```suggestion
description: Guided steps to write a zkApp that includes a smart contract and a UI. Create a smart contract with SnarkyJS for Mina Protocol.
```
